docs/get-started/alerting.md

```diff
@@ -6,64 +6,161 @@
+
+### Restore SMTP settings following an upgrade
+
+Upgrading PMM will overwrite the **grafana.ini** file with a newer version. If you configured PMM to use SMTP settings via environment variables, you do not need to do anything after an upgrade as your settings will be transferred.
+
+If the SMTP settings were configured directly in the **grafana.ini** file, make sure to manually back up the .ini file and copy your SMTP settings after the upgrade. This is because upgrading PMM will overwrite the grafana.ini file.
+
+### Configure an Email contact point
+
+After configuring the SMTP settings, specify email delivery options for an Email contact point: 
+
+1. Go to **Alerting > Contact points**.
+2. Click the edit button next to the **grafana-default-email** to update PMM's default Email contact point, or click **New contact point** to create a custom one.
+3. Enter a contact point name, and add the email addresses for the recipients of the email notifications.
+4. Expand **Optional settings** and fill in any other relevant settings:
+    - Enable the **Single email** option to send a single email to the recipients containing alerts that are firing. For example, if an alert fires for three nodes, this would send only one email listing all three alerts.
+    - Add an optional message to include with the email notifications.
+    - Edit the email subject for the notifications. The default subject line uses the following format [FIRING:*number of alerts firing for the alert rule*](*Name of alert rule and instance*).
+5. If you do not want to be notified when an alert resolves, expand **Notification settings**, and tick the **Disable Resolved Message** checkbox.
+6. If you want your contact point to notify via multiple channels, for example both via Email and Teams, click **New contact point type** and fill out additional contact point type details.
+7. Click the **Test** button to send a test email and make sure your contact point works as expected.
+8. Click the **Save contact point** button at the bottom of the page. The contact point is now listed under **Alerting > Contact points**.
+
+#### Create additional contact points
+
+In addition to Email contact points, you can add a variety of other contact points, including Slack, email, webhooks, PagerDuty, and more.
+
+Follow the steps above to create additional contact points. Different contact points require different configuration information. For example, for Slack, PMM requires the recipient information, the API token and the webhook URL, which you can get from your Slack administrator.
 
 ### Notification policies
-Notification policies determine how Grafana alerts are routed to contact points by setting where, when, and how to send notifications. 
+
+Notification policies determine how Grafana alerts are routed to contact points by defining where, when, and how to send notifications.
```

Comment:
change **alerts** to **notifications** - last comment on this (repeated)...you even have it as part of the sentence at the end...why confuse people needlessly?